build_config/lib/build_config.dart

```diff
@@ -13,4 +13,7 @@ export 'src/build_config.dart'
         TargetBuilderConfig,
         TargetBuilderConfigDefaults;
 export 'src/key_normalization.dart'
-    show normalizeBuilderKeyUsage, normalizeTargetKeyUsage;
+    show
+        normalizeBuilderKeyUsage,
+        normalizeTargetKeyUsage,
+        normalizeTargetKeyDefinition;
```

Comment:
I think we really just want `normalizeTargetKeyUsage` to understand this as opposed to exposing `normalizeTargetKeyDefinition`? 

The things we actually want to add support for are all usages I believe.

Reply:
Ah my bad, let me do that instead :)

Reply:
PTAL

Reply:
I can't find the usage of `normalize*` from apply_builders... did something change?